src/RestClient/endpoints/V3/Themes/Themes.ts

```diff
@@ -0,0 +1,61 @@
+import { AxiosInstance } from 'axios';
+
+import { AxiosPromise } from '../../../../types';
+import { buildThemeForm } from '../../../../utils/buildThemeForm';
+
+import type { Theme } from './types';
+
+export const themesPath = '/v3/themes';
+
+class Themes {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Gets a list of store themes
+   *
+   * @returns Promise resolving to a list of store themes
+   */
+  list(): Theme['ListResponse'] {
+    return this.client.get(themesPath);
+  }
+
+  /**
+   * Uploads a new theme to your BigCommerce store.
+   *
+   * @param themePath An absolute path to the theme file
+   * @returns Promise resolving with the job ID processing the theme upload
+   */
+  create(themePath: string): Theme['CreateResponse'] {
+    const themeForm = buildThemeForm(themePath);
```

Comment:
This method as well as buildImageForm should be refactored into a single method that takes two parameters. 

Something like:
```typescript
export const buildMultipartBody = (key: string, assetPath: string): FormData => {
  const multipartForm = new FormData();
  multipartForm.append(key, createReadStream(assetPath));

  return multipartForm;
};
```

Where the usage would be:
```typescript
// for themes
const themeForm = buildMultipartBody('file', themePath);

// for images
const imageForm = buildMultipartBody('image', imagePath);
```

Can you please refactor the two separate methods in `src/utils` into a single method similar to the one above, and then update and test `src/RestClient/endpoints/V3/Catalog/ProductModifierImages.ts`, `src/RestClient/endpoints/V3/Catalog/CategoryImages.ts`, and `src/RestClient/endpoints/V3/Catalog/BrandImages.ts`? 

In your unit tests, I would include both tests that current exist for added insurance that the multipart method works consistently across different file types.

TIA!